src/common/types.ts

```diff
@@ -160,5 +160,10 @@ export interface IPluginEvent {
   user: string;
 }
 
+export interface IEmojiTable {
+  emoji: string;
+  emojiValue: any; // This is what you will send to lambda
+}
```

Comment:
emojiValue should have a better name, likes `args` since its the args of a function being passed through